Assets/MRTK/SDK/Experimental/MixedRealityKeyboard/MRTKUGUIInputField.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    /// <summary>
+    /// A derived class of UGUI's InputField to workaround with some issues of typing on HoloLens 2
+    /// </summary>
+    public class MRTKUGUIInputField : InputField
+    {
+#if UNITY_2019_3_OR_NEWER
+        [SerializeField]
+        private bool disableUGUIWorkaround = false;
+
+        /// <summary>
+        /// Currently there is a Unity bug that needs a workaround. Please keep this setting to be false until an announcement of the version of Unity/UGUI that resolves the issue is made.
```

Comment:
Probably worth adding this as a `Tooltip` for the above private field as well.

Reply:
Hmm I decided to not put a tooltip there because input field has its own custom inspector so this bool does not show up in the inspector.